tests/e2e/virt_backup_restore_suite_test.go

```diff
@@ -44,32 +44,49 @@ type VmBackupRestoreCase struct {
 func runVmBackupAndRestore(brCase VmBackupRestoreCase, expectedErr error, updateLastBRcase func(brCase VmBackupRestoreCase), updateLastInstallTime func(), v *lib.VirtOperator) {
 	updateLastBRcase(brCase)
 
+	diskName := brCase.Name + "-disk"
+	vmName := brCase.Name + "-vm"
+
 	// Create DPA
 	backupName, restoreName := prepareBackupAndRestore(brCase.BackupRestoreCase, func() {})
 
 	err := lib.CreateNamespace(v.Clientset, brCase.Namespace)
 	gomega.Expect(err).To(gomega.BeNil())
 
-	// Create VM from clone of CirrOS image
-	err = v.CloneDisk(brCase.SourceNamespace, brCase.Source, brCase.Namespace, brCase.Name, 5*time.Minute)
+	// Create a standalone VM disk. This defaults to cloning the CirrOS image.
+	// This uses a DataVolume so import and clone progress can be monitored.
+	// Behavioral notes: CDI will garbage collect DataVolumes if they are not
```

Comment:
maybe better to add this notes direct to yaml?

Reply:
If there are multiple tests, it will be awkward to have this documentation in every one. The test code is doing a procedure that makes specific demands on the YAML, I think it makes more sense to document this procedure in that code, or maybe in a separate file under the docs directory.

I added a comment in the cirros-test-disk YAML, but I think the code comment should also still stay.